src/main/java/uk/gov/hmcts/reform/prl/services/SolicitorEmailService.java

```diff
@@ -64,15 +64,16 @@ public EmailTemplateVars buildEmail(CaseDetails caseDetails) {
                 .collect(Collectors.toList());
 
             String applicantNames = String.join(", ", applicantNamesList);
-
+            log.info("****************The Applicant/Applicants name/names we want to see for email is ***************: {}", applicantNames);
             Court court = null;
             court = courtLocatorService.getNearestFamilyCourt(caseData);
+            log.info("****************The court name we want to see for email is ***************: {}", court.getCourtName());
 
             return SolicitorEmail.builder()
                 .caseReference(String.valueOf(caseDetails.getId()))
                 .caseName(emailService.getCaseData(caseDetails).getApplicantCaseName())
                 .applicantName(applicantNames)
-                .courtName((court != null) ? court.getCourtName() : "")
+                .courtName(court.getCourtName())
```

Comment:
why changed this?

Reply:
Yeah reverted back to normal.Actually on monday the sonar showed error on this.Now its working normal.